### PR TITLE
Update include-mainpage-xaml.md to updated xaml code

### PR DESCRIPTION
**Pull request by 0xharkirat**
This code is changed to the code generated using the template. Unless it is causing a build error on the old code.

GitHub Issue (If applicable): closes #

## PR Type

What kind of change does this PR introduce?

- Documentation content changes

## What is the current behavior?

```xaml
<Page x:Class="Counter.MainPage"
      xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
      xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
      xmlns:local="using:Counter"
      Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
  <StackPanel HorizontalAlignment="Center"
              VerticalAlignment="Center">
    <TextBlock AutomationProperties.AutomationId="HelloTextBlock"
               Text="Hello Uno Platform"
               HorizontalAlignment="Center" />
  </StackPanel>
</Page>

```

It is causing a build error like this:
image

## What is the new behavior?

```xaml
<Page x:Class="Counter.Presentation.MainPage"
      xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
      xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
      xmlns:local="using:Counter.Presentation"
      xmlns:uen="using:Uno.Extensions.Navigation.UI"
      xmlns:utu="using:Uno.Toolkit.UI"
      xmlns:um="using:Uno.Material"
      NavigationCacheMode="Required"
      Background="{ThemeResource BackgroundBrush}">
  <StackPanel HorizontalAlignment="Center"
              VerticalAlignment="Center">
    <TextBlock AutomationProperties.AutomationId="HelloTextBlock"
               Text="Hello Uno Platform"
               HorizontalAlignment="Center" />
  </StackPanel>
</Page>

```

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ x] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ x] Validated PR `Screenshots Compare Test Run` results.
- [ x] Contains **NO** breaking changes
- [ x] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [ x] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):

### CODE REVIEW

**Comment by morning4coffe-dev** [comment]
Thank you for your contribution, @0xharkirat! Can you please reword (https://docs.github.com/en/pull-requests/committing-changes-to-your-project/creating-and-editing-commits/changing-a-commit-message) your commit to follow the Conventional commits standard? https://www.conventionalcommits.org/en/v1.0.0/ Thank you 😇

File: doc/articles/getting-started/counterapp/includes/include-mainpage-xaml.md [hunk 1]
```diff
@@ -1,16 +1,21 @@
...
-<Page x:Class="Counter.MainPage"
+<Page x:Class="Counter.Presentation.MainPage"
       xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
       xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
-      xmlns:local="using:Counter"
-      Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
+      xmlns:local="using:Counter.Presentation"
+      xmlns:uen="using:Uno.Extensions.Navigation.UI"
+      xmlns:utu="using:Uno.Toolkit.UI"
+      xmlns:um="using:Uno.Material"
```
Comment (Youssef1313) [inline doc/articles/getting-started/counterapp/includes/include-mainpage-xaml.md:10]: I don't see these namespaces being used.

**Comment by eriklimakc** [comment]
Hi @0xharkirat, I just created a new Uno app following the instructions in the Counter tutorial, and everything seems to be working correctly on my end. Could you double-check to ensure you used the `blank` preset rather than the `recommended` one? 

For reference the `MainPage` xaml from a newly created app with the latest template, using the `blank` preset.
```xaml
<Page x:Class="Counter.MainPage"
      xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
      xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
      xmlns:local="using:Counter"
      Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
  <ScrollViewer>
    <Grid>
      <StackPanel
        HorizontalAlignment="Center"
        VerticalAlignment="Center">
        <TextBlock AutomationProperties.AutomationId="HelloTextBlock"
          Text="Hello Uno Platform!"
          HorizontalAlignment="Center" />
      </StackPanel>
    </Grid>
  </ScrollViewer>
</Page>
```

**Comment by 0xharkirat** [comment]
Oh, I was creating the app from the recommended preset (not the blank one). I think then this request is unnecessary. I can close this one.

**Comment by agneszitte** [comment]
> Oh, I was creating the app from the recommended preset (not the blank one). I think then this request is unnecessary. I can close this one.

Hi @0xharkirat, thank you for the confirmation. Yes, the [Counter tutorial documentation](https://aka.platform.uno/counter-tutorial) mentions using the `blank` preset.
I am closing this pull request

**Pull request closed without merging**